src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -389,11 +389,12 @@ public BlockGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 				if (!isEditingMode()) {
 					lightOpacity.setValue(0);
+					lightOpacity.setEnabled(false);

```

Comment:
There's no need to disable light opacity

Reply:
If the light opacity can cause a problem we need to disable this parameter.

Reply:
This is intended behavior, added in #1463; the user has to manually change the light opacity for this to happen